pkg/piecestore/rpc/server/psdb/psdb.go

```diff
@@ -247,3 +274,75 @@ func (db *DB) DeleteTTLByID(id string) error {
 	}
 	return err
 }
+
+// AddMIB adds MIB into database by date
+func (db *DB) AddBwUsageTbl(size, unixtimenow int64) (err error) {
```

Comment:
Why is it using mebibytes instead of bytes?

`RecordBandwidthUsage(amount memory.Size, now time.Time) error`

Reply:
MIB was a term used with SNMP protocol days... It meant Management Information Base. We are not using it, will change the comment to reflect the functionality.